keywords/MobileRestClient.py

```diff
@@ -957,6 +957,8 @@ def add_conflict(self, url, db, doc_id, parent_revisions, new_revision, attachme
             doc["_attachments"] = {
                 atts[0].name: {"data": atts[0].data}
             }
+            if type(doc["_attachments"][attachment_name]["data"]) == bytes:
```

Comment:
if data is encrypted only this condition gets executed, as SG do not have this issue it will not effect the SG

Reply:
It is better to run upgrade test, as upgrade test uses cbl, it might use encrytion and it might fail. Anyways, it is always better to make sure existing tests are not broken as it common utils. There are many times we think it won't break, but breaks the code. Better to run to be safe side
you can run this job, http://uberjenkins.sc.couchbase.com:8080/job/sync-gateway-upgrade-centos-testserver/1467/ with your branch